step-controller/step-controller-server/src/main/java/step/core/plans/PlanPlugin.java

```diff
@@ -123,5 +128,18 @@ private void createTableSettingsIfNecessary(GlobalContext context, ScreenInput n
 		}
 	}
 
+	public static class PlanExportBiConsumer implements BiConsumer<Object, ExportContext> {

```

Comment:
Rename this class to a more meaningful name, maybe like MakeInvisiblePlansVisibleExportHook or similar

Reply:
Renamed the class but still keeping it bit generic since this is not reserved to plan visibility and could be extended with further modifications of plan during export